concrete/src/View/View.php

```diff
@@ -206,11 +206,11 @@ protected function loadViewThemeObject()
             switch ($this->themeHandle) {
                 case VIEW_CORE_THEME:
                     $this->themeObject = new \Concrete\Theme\Concrete\PageTheme();
-                    $this->pkgHandle = false;
+                    $this->themePkgHandle = false;
```

Comment:
It's correct that we have `themePkgHandle` instead of `pkgHandle`, isn't it?

Reply:
Yes, I believe so.